python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -98,8 +98,14 @@ class SensorType(Enum):
     MULTI_ASSET = "MULTI_ASSET"
     FRESHNESS_POLICY = "FRESHNESS_POLICY"
     AUTO_MATERIALIZE = "AUTO_MATERIALIZE"
+    ASSET_AUTOMATION = "ASSET_AUTOMATION"
```

Comment:
[1]

Reply:
Let's just call it the `AutomationConditionSensor` for now.